jobs/web/spec

```diff
@@ -1092,7 +1131,7 @@ properties:
   vault.path_prefix:
     env: CONCOURSE_VAULT_PATH_PREFIX
     description: |
-      Path under which to namespace team/pipeline credentials.
+      Path under which to look up shared team/pipeline credentials.
```

Comment:
smart. I noticed this in the Helm packaging (and the CLI flag) and didn't bother with it, but this is the Right Thing.